tox.ini

```diff
@@ -1,14 +1,14 @@
 [tox]
-envlist=py{27,35,36}
+envlist=py3
 skip_missing_interpreters=true
 
 [testenv]
 deps =
     -rdev-requirements.txt
 usedevelop=true
 commands =
-    pytest -vvs --cov=gdcmodels --cov-report xml --cov-report html --junit-xml test-reports/results.xml  {posargs}
     python ./scripts/git-hooks/pre-commit
+    pytest -vvs --cov=gdcmodels --cov-report xml --cov-report html --junit-xml test-reports/results.xml  {posargs}
```

Comment:
I agree that it's better to do this second.

We should move the YAML sorting into the pre-commit framework and then run `pre-commit run -a` here instead, so that we apply all of the checks. I can't remember if I already filed a ticket to make a YAML sorter that works with pre-commit. Anyway, that's way outside the scope of this ticket.

Reply:
maybe we can use one from (here)[https://pre-commit.com/hooks.html]?